contracts/governance/extensions/GovernorExtendedVoting.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts vX.X.X (governance/extensions/GovernorExtendedVoting.sol)
```

Comment:
Just so you know, this is added automatically so it's not necessary to include it manually in a PR.